ArcGISRuntimeSDKQt_CppSamples/Layers/CreateAndSaveKmlFile/CreateAndSaveKmlFile.h

```diff
@@ -68,16 +69,22 @@ class CreateAndSaveKmlFile : public QObject
   Esri::ArcGISRuntime::Point m_point;
   Esri::ArcGISRuntime::Polyline m_polyline;
   Esri::ArcGISRuntime::Polygon m_polygon;
+  Esri::ArcGISRuntime::KmlStyle* m_pointStyle = nullptr;
+  Esri::ArcGISRuntime::KmlStyle* m_lineStyle = nullptr;
+  Esri::ArcGISRuntime::KmlStyle* m_polygonStyle = nullptr;
 
   bool m_busy = false;
 
   Esri::ArcGISRuntime::Geometry createPoint() const;
   Esri::ArcGISRuntime::Geometry createPolyline() const;
   Esri::ArcGISRuntime::Geometry createPolygon() const;
   Esri::ArcGISRuntime::Geometry createEnvelope() const;
+  Esri::ArcGISRuntime::KmlStyle* createPointStyle();
+  Esri::ArcGISRuntime::KmlStyle* createLineStyle();
+  Esri::ArcGISRuntime::KmlStyle* createPolygonStyle();
```

Comment:
Consider renaming these methods. A KmlPolygonStyle is not a KmlStyle. A KmlStyle has-a KmlPolygonStyle. The method names could add to confusion.

How about these?
```c++
  Esri::ArcGISRuntime::KmlStyle* createKmlStyleWithPointStyle();
  Esri::ArcGISRuntime::KmlStyle* createKmlStyleWithLineStyle();
  Esri::ArcGISRuntime::KmlStyle* createKmlStyleWithPolygonStyle();
```

They're wordier, but I think they better communicate the object model and what's happening.